README.MD

```diff
@@ -0,0 +1,326 @@
+});
+```
+
+
+##### Deploy Organization contract
+
+An Organization contract serves as an on-chain access control mechanism by assigning roles to a set of addresses.
+
+```js
+// Deploy Organization contract
+const OrganizationHelper = Setup.OrganizationHelper;
+const organizationHelperObject = new OrganizationHelper(web3Provider);
+const organizationConfig = {
+  deployer: deployerAddress,
+  owner: organizationOwner,
+  workers: worker,
+  workerExpirationHeight: '20000000' // High expiration limit
+};
+let deployParams = {
+  gas: '7500000',
+  gasPrice: '0x3B9ACA00'
+};
+let organizationAddress;
+organizationHelperObject.setup(organizationConfig, deployParams).then(function(){
+    organizationAddress = organizationHelperObject.address;
+});
+
+
+```   
+
+##### Deploy BrandedToken contract
+BrandedToken is a value-backed EIP-20 token with a fixed conversion rate against the Value Token chosen. This contract is deployed on the value chain.
+```js
+const brandedTokenHelperConfig = {
+    deployer: deployerAddress,
+    valueToken: eip20Token,
+    symbol: 'BT',
+    name: 'MyBrandedToken',
+    decimals: '18',
+    conversionRate: '10',
+    conversionRateDecimals: 5,
+    organization: organizationAddress
+};
+deployParams = {
+  gas: '7500000',
+  gasPrice: '0x3B9ACA00'
+};
+const brandedTokenHelper = Setup.BrandedTokenHelper;
+let brandedTokenAddress;
+brandedTokenHelper.setup(brandedTokenHelperConfig, deployParams).then(function(){
+    brandedTokenAddress = brandedTokenHelper.address;
+});
+
+
+```  
+
+##### Deploy UtilityBrandedToken contract
+The UtilityBrandedToken is a representation of the BrandedToken on the sidechains. Thus, this contract is deployed on the sidechain
+
+```js
+const utilityBrandedTokenHelperConfig = {
+    deployer: deployerAddress,
+    token: brandedTokenAddress,
+    symbol: "UBT",
+    name: "MyBrandedToken",
+    decimals: "18",
+    organization: organizationAddress
+};
+deployParams = {
+  gas: '7500000',
+  gasPrice: '0x3B9ACA00'
+};
+const utilityBrandedTokenHelper = Setup.UtilityBrandedTokenHelper;
+let utilityBrandedTokenAddress;
+utilityBrandedTokenHelper.setup(utilityBrandedTokenHelperConfig, deployParams).then(function(){
+    utilityBrandedTokenAddress = utilityBrandedTokenHelper.address;
+});
+
+
+```
+
+##### Deploy GatewayComposer contract
+Gateway composer is a contract optimizes the transactions required to perform the stake and mint process. This contract is deployed on the value chain.
+
+```js
+const gatewayComposerConfig = {
+    deployer: deployerAddress,
+    owner: staker,
+    valueToken: eip20Token,
+    brandedToken: brandedTokenAddress
+};
+deployParams = {
+  gas: '7500000',
+  gasPrice: '0x3B9ACA00'
+};
+const gatewayComposerHelper = Setup.GatewayComposerHelper;
+let gatewayComposerAddress;
+gatewayComposerHelper.setup(gatewayComposerConfig, deployParams).then(function(){
+    gatewayComposerAddress = gatewayComposerHelper.address;
+});
+
+```
+
+##### Deploy Gateway Mock contract
```

Comment:
Please clarify whether a mock gateway contract is required or a real gateway contract may also be used.

Reply:
@abhayks1

Reply:
Appended below statement:
"Please skip this step if real Gateway contract address is available."